src/main/java/uk/org/tombolo/execution/DataExportEngine.java

```diff
@@ -101,4 +103,38 @@ private void importDatasource(ImporterMatcher forceImports, DatasourceRecipe dat
 				forceImports.doesMatch(datasourceSpec.getImporterClass())
 		);
 	}
+
+	private Importer initialiseImporter(String importerClass, String configFile) throws Exception {
+		Config importerConfiguration = null;
+		if (configFile != null && !"".equals(configFile)) {
+			importerConfiguration = ConfigUtils.loadConfig(
+					AbstractRunner.loadProperties("Configuration file", configFile));
+
+		}
+		return (Importer) Class.forName(importerClass).getDeclaredConstructor(Config.class).newInstance(importerConfiguration);
+	}
+
+	private String verifyProvider() throws Exception {
```

Comment:
You might consider adding this method and/or check in  DataExportRunner.java where the other validators are.

Reply:
There are 2 reasons why i didn't do it there,
1) DataExportRunner.java does the initial checks for arguments and structure of recipe, nothing related to importers or anything else. Thus i thought of leaving it the way it is, to have it confined till initial checks.
2) If we do that in DataExportRunner.java we have to repeat the initialiseImporters code and create a dummy variable for Config class, which i think is not good as creating dummy variables are usually an approach taken while creating test.